### PR TITLE
feat: Readme setup

### LINKED ISSUE #3397: feat: Add command to copy database template in readme setup
**Is your feature request related to a problem? Please describe.**
A lot of new contributors (especially one's new to rails) find the 4th step of the Setup.md readme file difficult to understand.

We can add the command directly they can use to copy the postgres credentials in the database.yml file

**Describe the solution you'd like**
add this command in readme

```bash
cp config/database.example.yml config/database.yml
```

**Describe alternatives you've considered**

**Additional context**
BEFORE
before
AFTER
after

**Are you working on this? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by VaibhavUpreti**
Fixes #3397

#### Describe the changes you have made in this PR -
added command to setup postgres in setup.md markdown file

### Screenshots of the changes (If any) -
after

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by VaibhavUpreti** [comment]
@tachyons @vedant-jain03  WDYT?

File: SETUP.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ cd CircuitVerse
 1. Install Ruby bundler : `gem install bundler`
 2. Install Ruby dependencies: `bundle install`
 3. Install Yarn dependencies: `yarn`
-4. Configure your PostgreSQL database in `config/database.yml` (copy `config/database.example.yml` for the template)
+4. Configure your PostgreSQL database in `config/database.yml` (copy `config/database.example.yml` for the template): 
+     * **command:** `cp config/database.example.yml config/database.yml`
```
Comment (tachyons) [inline SETUP.md:67]: This command work in windows

Comment (VaibhavUpreti) [reply SETUP.md:67]: this works on mac too(tried) , also works on linux with -r flag

Comment (tachyons) [reply SETUP.md]: Oops, I made a typo

**Pull request merged**